test/acceptance/docker-compose.override.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
+services:
+  wdio-cucumber:
+    volumes:
+      - ./:/home/node/src
+      - ./babel.config.js:/home/node/babel.config.js
+      - ./wdio.conf.js:/home/node/wdio.conf.js
+      - ./wdio.BUILD.conf.js:/home/node/wdio.BUILD.conf.js
+      - ./reports/html-reports/:/home/node/reports/html-reports/
```

Comment:
this could be mapped to an `html-reports` folder locally to avoid the `reports/html-reports` hierarchy

Reply:
OK, changed in latest commit